var/spack/repos/builtin/packages/gdbm/package.py

```diff
@@ -45,4 +45,25 @@ class Gdbm(AutotoolsPackage):
     depends_on("readline")
 
     def configure_args(self):
-        return ['--enable-libgdbm-compat']
+        spec = self.spec
+
+        # Environment variables
+        CFLAGS   = []
+        CPPFLAGS = []
+        LDFLAGS  = []
+        LIBS     = []
+
+        CPPFLAGS.append("-I%s/include" % spec['readline'].prefix)
+        LDFLAGS.append("-L%s/lib"     % spec['readline'].prefix)
+        LIBS.append("-l%s"         % "readline")
```

Comment:
This is fine and the compiler wrappers don't add it -- I guess the build system doesn't either.

Reply:
We don't need the `-l`, that actually already worked. I just put all these in, as a template.